src/main/java/org/ow2/proactive/catalog/service/BucketService.java

```diff
@@ -173,11 +173,13 @@ public BucketMetadata getBucketMetadata(long id) {
         return new BucketMetadata(bucket);
     }
 
-    public List<BucketMetadata> listBuckets(Optional<String> ownerName) {
+    public List<BucketMetadata> listBuckets(Optional<String> ownerName, Optional<String> kind) {
         List<BucketEntity> entities;
 
         if (ownerName.isPresent()) {
```

Comment:
I didn't see the use of filtering according to both fields. So if both are present only the user will be taken into account. Do you think i should change this?